### ISSUE TITLE
PHP 8.3 Compatibility

### ISSUE THREAD

**Issue by joelpittet** [ubc-cpsc/phpicalendar#1]
`phpcs --standard=PHPCompatibility --runtime-set testVersion 8.1-8.3`

```
FILE: /Users/pittet/Projects/phpicalendar/lib/bennu/bennu.class.php
---------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
---------------------------------------------------------------------------------------------------------------------
 21 | WARNING | Function gmstrftime() is deprecated since PHP 8.1; Use date() or IntlDateFormatter::format() instead
---------------------------------------------------------------------------------------------------------------------


FILE: /Users/pittet/Projects/phpicalendar/lib/HTTP/WebDAV/Server.php
------------------------------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------------------------------------
 2377 | WARNING | Function utf8_encode() is deprecated since PHP 8.2; Use mb_convert_encoding(), UConverter::transcode() or iconv instead
------------------------------------------------------------------------------------------------------------------------------------------


FILE: /Users/pittet/Projects/phpicalendar/templates/green/default.css
-----------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-----------------------------------------------------------------------------
 1 | WARNING | File has mixed line endings; this may cause incorrect results
-----------------------------------------------------------------------------


FILE: /Users/pittet/Projects/phpicalendar/templates/default/default.css
-----------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-----------------------------------------------------------------------------
 1 | WARNING | File has mixed line endings; this may cause incorrect results
-----------------------------------------------------------------------------


FILE: /Users/pittet/Projects/phpicalendar/templates/grey/default.css
-----------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-----------------------------------------------------------------------------
 1 | WARNING | File has mixed line endings; this may cause incorrect results
-----------------------------------------------------------------------------


FILE: /Users/pittet/Projects/phpicalendar/templates/red/default.css
-----------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-----------------------------------------------------------------------------
 1 | WARNING | File has mixed line endings; this may cause incorrect results
-----------------------------------------------------------------------------


FILE: /Users/pittet/Projects/phpicalendar/templates/tan/default.css
-----------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-----------------------------------------------------------------------------
 1 | WARNING | File has mixed line endings; this may cause incorrect results
-----------------------------------------------------------------------------

Time: 1.64 secs; Memory: 28MB

```

**Comment by joelpittet** [comment]
Fixed the unrelated CSS errors in 3566ff9

**Comment by joelpittet** [comment]
Fixed the other two in 597daba and 6b707a7
